### DISCUSSION TITLE
how to configure typedoc to create one large markdown file for injection into existing README.md

### DISCUSSION THREAD

**Discussion by airtonix** [tgreyuk/typedoc-plugin-markdown#577] [Help] [answered]
just want to inject the documentation into the existing `README.md`

**Comment by tgreyuk** [comment, marked as answer]
To create single API docs currently you need to export a single entry point. If you also want to keep merge it with your readme you can use `mergeReadme` which basically prepends it to the resolved readme file.

Here is an example:

~https://github.com/tgreyuk/typedoc-plugin-markdown-examples/tree/main/examples/01-typedoc-plugin-markdown/05-single-page-docs-with-readme~

https://github.com/typedoc2md/typedoc-plugin-markdown-examples

**Reply by bukowa** [reply]
Links are dead

**Reply by tgreyuk** [reply]
Link updated.
